src/applications/disability-benefits/526EZ/helpers.jsx

```diff
@@ -118,7 +120,7 @@ export const disabilityNameTitle = ({ formData }) => {
 
 export const facilityDescription = ({ formData }) => {
   return (
-    <p>Tell us about facilities where VA treated you for {getDiagnosticCodeName(formData.diagnosticCode)}, <strong>after you got your disability rating</strong>.</p>
+    <p>Please tell us where VA treated you for {getDiagnosticCodeName(formData.diagnosticCode)} <strong>after you got your disability rating</strong>.</p>
```

Comment:
[Not something to address in this PR]
`getDiagnosticCodeName` returns 'Unknown Condition' in the case that a mapping isn't available for the input diagnostic code. Maybe 'condition' would be more appropriate as default text. I realize this is an edge-case either way.